test/gaudi_opts/test_global_converter_maps.py

```diff
@@ -90,11 +90,7 @@
     )
 
 inputConverter = EDM4hep2LcioTool("InputConverter")
-inputConverter.convertAll = False
-inputConverter.collNameMapping = {
-    "MCParticles": "MCParticles",
-    "PseudoRecoParticles": "PseudoRecoParticles",
-}
+inputConverter.convertAll = True
```

Comment:
I guess this was failing before and would show up in the `TrivialMCTruthLinker` that it would miss some collections?

Reply:
Yes, this was failing before for the `PodioInput` version of the test, because we only read `EventHeader` and `MCParticles`, but before https://github.com/key4hep/k4FWCore/pull/290 the `Frame` in the `PodioDataSvc` would claim that it has all collections and then fail when they are not in the TES. Initially this was an optimization to make the test quicker to run by only reading and converting what is strictly necessary.